ui/raidboss/data/07-dt/dungeon/skydeep-cenote.ts

```diff
@@ -0,0 +1,362 @@
+        },
+        stackFive: {
+          en: 'Stack (5 hits)',
+        },
+      },
+    },
+    {
+      id: 'SkydeepCenote Maulskull Initial Stonecrusher',
+      type: 'StartsUsingExtra',
+      // 8F3E is the first half-arena cleave, but the actor can be east or west.
+      netRegex: { id: '8F3E' },
+      durationSeconds: 10,
+      alertText: (_data, matches, output) => {
+        const firstCleave = parseFloat(matches.x) > 100 ? 'east' : 'west';
+        return output[firstCleave]!();
+      },
+      outputStrings: {
+        east: Outputs.leftThenRight,
+        west: Outputs.rightThenLeft,
+      },
+    },
+    {
+      id: 'SkydeepCenote Maulskull Initial Skullcrush',
+      type: 'StartsUsing',
+      netRegex: { id: '8F43', source: 'Maulskull', capture: false },
+      durationSeconds: 6,
+      infoText: (_data, _matches, output) => output.kbAoeSpread!(),
+      outputStrings: {
+        kbAoeSpread: {
+          en: 'Knockback (AoE) => Spread',
+        },
+      },
+    },
+    {
+      id: 'SkydeepCenote Maulskull Maulwork + Shatter Middle',
+      type: 'Ability',
+      netRegex: { id: ['8F47', '8F48'], source: 'Maulskull', capture: false },
+      durationSeconds: 9,
+      infoText: (_data, _matches, output) => output.dodgeSides!(),
+      outputStrings: {
+        dodgeSides: {
+          en: 'Avoid AoEs (sides after)',
+        },
+      },
+    },
+    {
+      id: 'SkydeepCenote Maulskull Maulwork + Shatter Middle Followup',
+      type: 'Ability',
+      netRegex: { id: ['8F47', '8F48'], source: 'Maulskull', capture: false },
+      delaySeconds: 10,
+      response: Responses.goSides(),
+      run: (data) => data.seenShatter = true,
+    },
+    {
+      id: 'SkydeepCenote Maulskull Maulwork + Shatter Sides',
+      type: 'Ability',
+      netRegex: { id: ['8F49', '8F4A'], source: 'Maulskull', capture: false },
+      durationSeconds: 9,
+      infoText: (_data, _matches, output) => output.dodgeMiddle!(),
+      outputStrings: {
+        dodgeMiddle: {
+          en: 'Avoid AoEs (middle after)',
+        },
+      },
+    },
+    {
+      id: 'SkydeepCenote Maulskull Maulwork + Shatter Sides Followup',
+      type: 'Ability',
+      netRegex: { id: ['8F49', '8F4A'], source: 'Maulskull', capture: false },
+      delaySeconds: 10,
+      response: Responses.goMiddle(),
+      run: (data) => data.seenShatter = true,
+    },
+    {
+      id: 'SkydeepCenote Maulskull Stonecrusher + Skullcrush',
+      type: 'StartsUsingExtra',
+      // 8F58 is the first half-arena cleave, but the actor can be east or west.
+      netRegex: { id: '8F58' },
+      durationSeconds: 11,
+      alertText: (_data, matches, output) => {
+        const firstCleave = parseFloat(matches.x) > 100 ? 'east' : 'west';
+        return output[firstCleave]!();
+      },
+      outputStrings: {
+        east: {
+          en: '<== Knockback Back Left (Right After)',
+        },
+        west: {
+          en: 'Knockback Back Right ==> (Left After)',
```

Comment:
```suggestion
          en: '<= Knockback Back Left (Right After)',
        },
        west: {
          en: 'Knockback Back Right => (Left After)',
```
The long arrow `==>` is causing TTS to give weird output here.

Reply:
Ahh, it looks like TTS will only strip leading/trailing arrows if they have a single `=` or `-`; otoh, a `=>` in the middle of the string is output as `then`:
https://github.com/OverlayPlugin/cactbot/blob/27eccdb577e18faf1847325bb76f8ca382573b64/ui/raidboss/popup-text.ts#L1544-L1558

('then' was not actually what I had intended with KB Right ==> (Left After) call)

For safety, I'm wondering whether it might make more sense to have TTS strip out arrows at the beginning/end of a string regardless of the number of `-` or `=` are in the arrow?  (No real good reason why `==>` should be treated differently than `=>`.  And the second callout should probably just have the arrow moved to the end of the string.  Thoughts?

Reply:
I think in the immediate case, moving the arrow to the end makes sense, since that fits usage elsewhere. I never use TTS myself, but as a non-user it makes sense to me to strip arrows that are are the start or end of the string.

Reply:
> Ahh, it looks like TTS will only strip leading/trailing arrows if they have a single `=` or `-`; otoh, a `=>` in the middle of the string is output as `then`:
> 
> https://github.com/OverlayPlugin/cactbot/blob/27eccdb577e18faf1847325bb76f8ca382573b64/ui/raidboss/popup-text.ts#L1544-L1558
> 
> ('then' was not actually what I had intended with KB Right ==> (Left After) call)
> 
> For safety, I'm wondering whether it might make more sense to have TTS strip out arrows at the beginning/end of a string regardless of the number of `-` or `=` are in the arrow? (No real good reason why `==>` should be treated differently than `=>`. And the second callout should probably just have the arrow moved to the end of the string. Thoughts?

TTS already effectively ignores `=>` and `<=` at the beginning or end of a string (nothing gets spoken); enforcing that explicitly shouldn't change how any existing calls work, beyond allowing `==>`, `===>`, etc without unexpected output.